docs/ref/databases.txt

```diff
@@ -367,28 +367,6 @@ this for you (the database backend layer and the model population layer are
 separated internally so the database layer doesn't know it needs to make this
 conversion in this one particular case).
 
-If you're using MySQLdb 1.2.1p2, Django's standard
```

Comment:
Also @ramiro might know something about this (bb10a5ba79ecaea7d32b62afe90e83407adf0b74)?